figures/models.py

```diff
@@ -206,11 +206,11 @@ class LearnerCourseGradeMetrics(TimeStampedModel):
     actually needed and edx-platform uses FloatField in its grades models
 
     """
-    site = models.ForeignKey(Site)
-    date_for = models.DateField()
+    site = models.ForeignKey(Site, db_index=True)
```

Comment:
As far as I know `ForeignKey` always implies db_index at the SQL level. [Stackoverflow agrees](https://stackoverflow.com/a/304339).

```suggestion
    site = models.ForeignKey(Site)
```